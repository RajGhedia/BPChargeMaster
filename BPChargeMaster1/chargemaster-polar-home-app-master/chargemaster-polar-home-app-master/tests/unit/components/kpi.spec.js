import Kpi from '@/components/kpi.vue'
import Vue from 'vue'
import Vuetify from 'vuetify'
import { mount, createLocalVue } from '@vue/test-utils'

const localVue = createLocalVue()
Vue.use(Vuetify)

describe('kpi.vue', () => {
    it('renders correctly when given props', () => {
        const wrapper = mount(Kpi, {
            localVue,
            propsData: {
                icon: 'total-charges-icon',
                description: 'a label',
                amount: '4.25'
            }
        })

        expect(wrapper.element).toMatchSnapshot()
    })
})
