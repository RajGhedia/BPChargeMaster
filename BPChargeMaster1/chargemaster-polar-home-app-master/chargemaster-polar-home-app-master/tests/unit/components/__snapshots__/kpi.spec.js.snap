// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`kpi.vue renders correctly when given props 1`] = `
<div
  class="layout kpi mb-3"
>
  <div
    class="flex xs8 offset-xs1 pt-3"
  >
    <div
      class="headline ma-0 pa-0"
    >
      4.25
    </div>
     
    <p
      class="subheading pa-0 mb-2"
    >
      a label
    </p>
  </div>
   
  <div
    class="flex xs4 pt-1 pr-3"
  >
    <div
      class="v-responsive v-image icon"
    >
      <div
        class="v-responsive__content"
      />
    </div>
  </div>
</div>
`;
